app/controllers/finance/api/invoices_controller.rb

```diff
@@ -6,6 +6,8 @@ class Finance::Api::InvoicesController < Finance::Api::BaseController
 
   STATES = Hash.new {|_, k| k}.merge('cancel' => 'cancelled').freeze
 
+  before_action :authorize_finance
```

Comment:
Shouldn't we add to `Finance::Api::PaymentTransactionsController` too? Why not adding to `Finance::Api::BaseController` then?